databricks-iris/{{ cookiecutter.repo_name }}/README.md

```diff
@@ -6,6 +6,53 @@ This is your new Kedro project, which was generated using `kedro {{ cookiecutter
 
 Take a look at the [Kedro documentation](https://docs.kedro.org) to get started.
 
+## Getting Started
+
+### Installing dependencies
+
+To install the dependencies, navigate to the project root and run:
+
+```sh
+# Activate the virtual environment
+python -m venv .venv
+source .venv/bin/activate
+
+# Install the dependencies
+pip install --upgrade pip
+pip install -r requirements.txt
+```
+
+### Initialization
+
+Initialize your Kedro project for Databricks with:
+
+```sh
+kedro databricks init
+```
+
+This command will create essential configuration files, including databricks.yml for Asset Bundle settings and conf/base/databricks.yml for workflow overrides. Customize these files to tailor the integration to your specific needs.
+
+### Generation
+
+Generate the Asset Bundle resources definitions using:
+
+```sh
+kedro databricks bundle
+```
+
+This will produce configuration files that map to your Kedro pipelines, readying them for execution on Databricks.
+
+### Deployment
+
+Deploy your Kedro project to Databricks seamlessly with:
+
+```sh
+kedro databricks deploy
```

Comment:
I get an error like this 
```
    raise FileNotFoundError(
FileNotFoundError: Configuration file /Users/Nok_Lam_Chan/tmp/iris/databricks.yml does not exist
```

Reply:
I have updated the help message to be a bit more informative 👍